### PR TITLE
feat: upload trained model to hf

### PR DESCRIPTION
**Pull request by l45k**
This will upload a trained model to a Hugging Face repositry if the following is porvided in the Schedulers' `conifg.toml`

```toml
[scheduler.job.model_destination]
repositry = "repositry/identifier"
token = "hf_*"
```

### CODE REVIEW

File: docs/content/scheduler.md [hunk 1]
```diff
@@ -118,6 +118,19 @@ learning_rate = 0.7
...
+**Upload Trained Model To HF**:
+To upload the trained model to Hugging Face provide a repository and a token with with the correct rights:
+
+```toml
+[scheduler.job.model_destination]
+repositry = "repositry/identifier"
+token = "hf_*"
+```
+
+> [!NOTE]
```
Comment (orlandohohmeier) [inline docs/content/scheduler.md]:
```suggestion
> [!IMPORTANT]
```

This feels kinda important... let's use the resp. flag

File: docs/content/scheduler.md [hunk 2]
```diff
@@ -118,6 +118,19 @@ learning_rate = 0.7
 momentum = 0.9
 ```
 
+**Upload Trained Model To HF**:
+To upload the trained model to Hugging Face provide a repository and a token with with the correct rights:
+
+```toml
+[scheduler.job.model_destination]
+repository="repository/identifier"
+token="hf_*"
```
Comment (orlandohohmeier) [inline docs/content/scheduler.md:133]: Let's add a note/warning that the token is going to be shared with the workers - same warning should be added to using a token for fetching a model.

**Review by orlandohohmeier — DISMISSED** [review]

**Review by orlandohohmeier — APPROVED** [review]
